r_markdown/emissions.Rmd

```diff
@@ -261,7 +261,7 @@ Select co, co2 and ch4.
 
 ```{r, echo=TRUE, warning=FALSE, message=FALSE, results='markup', cache=FALSE}
   co_co2_ch4 <- dplyr::group_by(co_co2_ch4, id, pol) %>%
-                dplyr::summarise(ppm = mean(ppm, na.rm = TRUE)) %>%
+                dplyr::summarise(ppm = mean(ppm, na.rm = TRUE), qc = first(qc)) %>%
```

Comment:
all qc comments should be the same for real-time instruments so just grabbing the first one and linking that to the summary file